yarn.lock

```diff
@@ -10873,7 +10873,7 @@ stylelint-order@^5.0.0:
     postcss "^8.3.11"
     postcss-sorting "^7.0.1"
 
-stylelint-scss@^4.1.0, stylelint-scss@^4.6.0:
+stylelint-scss@^4.1.0:
```

Comment:
note to self - we still have this major version because stylelint-config-palantir depends on it, and that package has not been updated in a while.